aws/v19.1.0/README.md

```diff
@@ -0,0 +1,267 @@
+
+## AWSMachineDeployment CR's annotation to change the Flatcar Release Version
+
+This feature allows customers to set an annotation on AWSMachineDeployment CR's to change the Flatcar Release Version. For now it only allows setting `alpha.giantswarm.io/flatcar-release-version: "3689.0.0"` or higher version. We have added this feature to accommodate the issues with Cilium CNI high CPU usage on small clusters. This feature is solely to enable customers to run the Flatcar `alpha` channel which consists of the `kernel 6` version that fixes the issue, while waiting for a `stable` Flatcar release.
+
+The annotation behaves as follows:
+- when setting the annotation, the TCNP CloudFormation Stack for the specific node pool is rolled and replaces the OS image
+- when removing the annotation, the node pool is updated and switches back to the default OS image which is coming from the AWS release
+- when upgrading the cluster to a new AWS release, the node pool uses the specific flatcar release from the annotation as long as you don't change by either setting it to a higher version or removing the annotation.
+
+## Change details
+
+
+### etcd [3.5.9](https://github.com/etcd-io/etcd/releases/tag/v3.5.9)
+
+#### etcd server
+- Fix [LeaseTimeToLive API may return keys to clients which have no read permission on the keys](https://github.com/etcd-io/etcd/pull/15815).
+#### Dependencies
+- Compile binaries using [go 1.19.9](https://github.com/etcd-io/etcd/pull/15822).
+
+
+
+### kubernetes [1.24.17](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.17)
+
+#### Feature
+- Kubernetes is now built with Go 1.20.7 ([#119837](https://github.com/kubernetes/kubernetes/pull/119837), [@jeremyrickard](https://github.com/jeremyrickard)) [SIG Apps, Cloud Provider, Node, Release, Storage and Testing]
+#### Bug or Regression
+- Fixed a bug where clusters that use KMS v1 with skewed API servers on versions v1.24 and v1.25 would see internal errors when attempting to read encrypted data via the v1.24 API servers. ([#119387](https://github.com/kubernetes/kubernetes/pull/119387), [@enj](https://github.com/enj)) [SIG API Machinery and Auth]
+#### Dependencies
+#### Added
+_Nothing has changed._
+#### Changed
+_Nothing has changed._
+#### Removed
+_Nothing has changed._
+
+
+
+### app-operator [6.8.0](https://github.com/giantswarm/app-operator/releases/tag/v6.8.0)
+
+#### Added
+- Add Service Monitor by default to make it complain with the latest monitoring improvements 
+
+
+
+### aws-operator [14.22.0](https://github.com/giantswarm/aws-operator/releases/tag/v14.22.0)
+
+#### Changed
+
+- Get AMI data from helm value rather than from hardcoded string in the code.
+
+
+
+⚠️ Attention: Major release [3.0.0](#300---2023-07-26) contains breaking changes in user values! Please make sure you [read about the upgrade instructions](https://github.com/giantswarm/cert-manager-app/blob/main/docs/upgrading.md)! ⚠️
```

Comment:
why do we have `cert-manager` stuff under `aws-operator`?

Reply:
fixed, maybe smbd messed up copy paste